src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -7,6 +7,33 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Removed the following package references:
+
+  * `Google.Protobuf`
+  * `Grpc`
+  * `Grpc.Net.Client`
+  * `Grpc.Tools`
+
+  These changes were made to streamline dependencies and reduce the footprint of
+  the exporter.
+  ([#6015](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6015))
```

Comment:
I think that, you have wrong links in most cases. It refers to the adding changelog entry, instead of real changes.
One entry could have more than one link.